### ISSUE TITLE
error on stop

### ISSUE THREAD

**Issue by jan-bogaerts** [Tiemooowh/pyTeletask#2]
https://github.com/Tiemooowh/pyTeletask/blob/2a547b4ca9b0a498c5f72c4de61a7423eb727690/teletask/io/client.py#L132

Hi, 
I think this line has an error (at least I'm getting errors on it). 
Maybe it should be?
```
self.writer.close()
```

**Comment by Tiemooowh** [comment]
Looks indeed reasonable to me that this will cause errors (probably the
writer line?)

**Comment by jan-bogaerts** [comment]
yes. I eventually commented it out (there was no close). Not certain if it has value.

**Comment by Tiemooowh** [comment]
Don't really bring that much to the table, since you'll probably want it
"always running" in some kind of token-loop ;-)

T
